flows/waits/time.go

```diff
@@ -0,0 +1,25 @@
+package waits
+
+import (
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/flows/events"
+)
+
+const TypeTime string = "time"
+
+type TimeWait struct {
+	Timeout int `json:"timeout" validation:"required,gte=0"`
+}
+
+func (w *TimeWait) Type() string { return TypeMsg }
+
+func (w *TimeWait) Apply(run flows.FlowRun, step flows.Step) {
+	run.ApplyEvent(step, nil, &events.TimeWaitEvent{Timeout: w.Timeout})
+}
+
+// CanResume returns true for a message wait if a message has now been received on this step
+func (w *TimeWait) CanResume(run flows.FlowRun, step flows.Step) bool {
+	return true
```

Comment:
I don't think it's goofy to have a special case of a wait that is essentially a null wait - and it provides an easy mechanism to let callers commit sessions before a potentially long running action.

If we return the same wait the a caller will wait for that time period again, which isn't correct.